Modules/LearningSequence/classes/class.ilObjLearningSequenceContentGUI.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+require_once("./Services/Form/classes/class.ilPropertyFormGUI.php");
+require_once("./Modules/LearningSequence/classes/class.ilObjLearningSequence.php");
+
+/**
+ * Class ilObjLearningSequenceContentGUI class
+ */
+class ilObjLearningSequenceContentGUI
+{
+
+	public function __construct(
+		\ilObjLearningSequenceGUI $a_parent_gui,
+		int $a_ref_id,
+		\ILIAS\DI\Container $dic,
+		\ilLanguage $lng
+	)
+	{
```

Comment:
public function __construct(
		\ilObjLearningSequenceGUI $a_parent_gui,
		int $a_ref_id,
		\ILIAS\DI\Container $dic,
		\ilLanguage $lng
	) {
		$this->ctrl = $dic->ctrl();
	}